hazelcast/util.py

```diff
@@ -219,3 +219,27 @@ def get_portable_version(portable, default_version):
     except AttributeError:
         version = default_version
     return version
+
+
+class TimeUnit(object):
+    """
+    Represents the time durations at given units in seconds.
+    """
+    NANOSECOND = 1e-9
+    MICROSECOND = 1e-6
```

Comment:
I think it is better to express the units with respect to each other. For example instead of `MICROSECOND = 1e-6`, we can use `MICROSECOND = NANOSECOND *1000` and so on.